README.md

```diff
@@ -20,4 +20,4 @@ Please [open a discussion](https://github.com/observablehq/framework/discussions
 
 ## Contributing 🙏
 
-See [`CONTRIBUTING.md`](./CONTRIBUTING.md).
+See [`docs/contributing.md`](./docs/contributing.md).
```

Comment:
```suggestion
See [Contributing](https://observablehq.com/framework/contributing).
```

probably better to link to the site?

Reply:
That works for me if that's what you all prefer.

I suppose the tradeoff is better styling on the observable site versus staying within the GitHub project where you may already be working.